src/coreclr/gc/gc.cpp

```diff
@@ -8519,10 +8523,12 @@ void gc_heap::get_card_table_element_sizes (uint8_t* start, uint8_t* end, size_t
     }
 #endif //CARD_BUNDLE
 #ifdef FEATURE_USE_SOFTWARE_WRITE_WATCH_FOR_GC_HEAP
+#ifdef BACKGROUND_GC
```

Comment:
should be combined -

`#if defined(FEATURE_USE_SOFTWARE_WRITE_WATCH_FOR_GC_HEAP) && defined (BACKGROUND_GC)`